### PR TITLE
feat: separate search and explore

### PR DESCRIPTION
**Pull request by patak-dev**
Closes #2064
Closes #1704

See @TodePond post for reference https://github.com/elk-zone/elk/pull/1301#issuecomment-1416490216
 
Partial revert of #1301

@shuuji3 it was easier to do the revert, please test this one. Thanks!

### CODE REVIEW

**Comment by TodePond** [comment]
My hero! This is a great digital well-being improvement for me! My prediction is that some people might not like this 🤔 Ideally we could put this behind an option, but I understand that that would come with its own cost!

**Comment by patak-dev** [comment]
Yes, lets add the option if there is a lot of demand. Even then I would push back to try to avoid it though.

**Review by shuuji3 — APPROVED** [review]
@patak-dev This is exactly what I was planning to change on #2064. Thanks for helping 🙂

File: components/nav/NavBottom.vue [hunk 1]
```diff
@@ -16,7 +16,7 @@ const { notifications } = useNotifications()
       <NuxtLink to="/home" :aria-label="$t('nav.home')" :active-class="moreMenuVisible ? '' : 'text-primary'" flex flex-row items-center place-content-center h-full flex-1 class="coarse-pointer:select-none" @click="$scrollToTop">
         <div i-ri:home-5-line />
       </NuxtLink>
-      <NuxtLink :to="isHydrated ? `/${currentServer}/explore` : '/explore'" :aria-label="$t('nav.explore')" :active-class="moreMenuVisible ? '' : 'text-primary'" flex flex-row items-center place-content-center h-full flex-1 class="coarse-pointer:select-none" @click="$scrollToTop">
+      <NuxtLink to="/search" :aria-label="$t('nav.explore')" :active-class="moreMenuVisible ? '' : 'text-primary'" flex flex-row items-center place-content-center h-full flex-1 class="coarse-pointer:select-none" @click="$scrollToTop">
```
Comment (shuuji3) [inline components/nav/NavBottom.vue:19]: It looks like the `aria-label` was not changed here:

```suggestion
      <NuxtLink to="/search" :aria-label="$t('nav.search')" :active-class="moreMenuVisible ? '' : 'text-primary'" flex flex-row items-center place-content-center h-full flex-1 class="coarse-pointer:select-none" @click="$scrollToTop">
```

Comment (patak-dev) [reply components/nav/NavBottom.vue:19]: Good catch!

**Pull request merged**